Readme.md

```diff
@@ -47,6 +47,12 @@ For .NET workflows, the default .NET SDK version we should provide is a concrete
 
 We can still choose to update to a more recent patch version, but only deliberately.
 
+### Default Node.js version
+
+.NET build workflows install the latest Active Long-Term Support (LTS) version of Node.js by default, but the version is configurable. You can check out the [Node.js release schedule](https://nodejs.org/en/about/previous-releases) to see which version is the latest Active LTS at any given time.
+
+Build workflows also enable [corepack](https://github.com/nodejs/corepack) to allow using Node.js package managers other than NPM. This behavior will be disabled when upgrading to Node.js v26 since that version [won't contain corepack anymore](https://github.com/nodejs/node/issues/50963#issuecomment-2746471823).
```

Comment:
Is this warning line necessary? It seems like after that point we can just replace `corepack enable` with `corepack enable to npm i -g corepack` so the users don't necessarily need to know about this. Or if you want to be fancy:

```pwsh
$nodeVersion = [version] $(node -v).Substring(1)

if ($nodeVersion.Major -ge 25)
{
    npm install --global corepack
}
else
{
    corepack enable
}
```

Reply:
Okay then.